modules/cnv-edit-boot-order-yaml-web.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+//
+
+[id="cnv-edit-boot-order-yaml-web_{context}"]
+= Editing a boot order list in the YAML configuration file
+
+Edit the boot order list in a YAML configuration file by using the CLI.
+
+.Procedure
+
+. Open the YAML configuration file for the virtual machine by running the following command:
++
+----
+$ oc edit vm example
+----
+
+. Edit the YAML file and modify the values for the boot order associated with a disk or Network Interface Card (NIC).For example:
+
++
+[source,yaml]
+----
+disks:
+  - bootOrder: 1 <1>
+    disk:
+      bus: virtio
+    name: containerdisk
+  - disk:
+      bus: virtio
+    name: cloudinitdisk
+  - cdrom:
+      bus: virtio
+    name: cd-drive-1
+interfaces:
+  - boot Order: 2 <2>
+    macAddress: '02:96:c4:00:00'
+    masquerade: {}
+    name: default
+----
+<1> The boot order value specified for the disk.
+<2> The boot order value specified for the Network Interface Card.
+
+. Save the YAML file. In the the web console, when you click the pencil icon next to *Boot Order* in the *Virtual Machine Overview* page, a message is displayed: *Boot order has been updated outside this flow. To see the updated order, reload the content.*
```

Comment:
Should this be two separate steps (saving the file and then clicking the pencil icon)? I can't tell how they're related.

Reply:
Saving the file and clicking the pencil icon are separate steps. Made the change.